.github/CODEOWNERS

```
@@ -1,2 +1 @@
-# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
-* @sonarsource/analysis-cloud-squad
+.github/CODEOWNERS * @sonarsource/analysis-cloud-squad
```

The PR shows that the file contains errors...
Maybe this can help (CamelCase):

```suggestion
.github/CODEOWNERS @SonarSource/analysis-cloud-squad
```

---

Thanks, there was a redundant `*`